pandas/tools/util.py

```diff
@@ -173,6 +174,8 @@ def to_numeric(arg, errors='raise', downcast=None):
             values = arg.values
     elif isinstance(arg, (list, tuple)):
         values = np.array(arg, dtype='O')
+    elif isinstance(arg, Decimal):
```

Comment:
also need to add some tests where decimals are in a list/array (they will be then subseuqnetly converted via ``maybe_convert_numeric`` which is also fine.